lib/smart_answer_flows/coronavirus-employee-risk-assessment/questions/is_your_workplace_an_exception.govspeak.erb

```diff
@@ -1,22 +1,47 @@
+<%
+  render_terse_options =  false
+  workplace =  calculator.where_do_you_work
+  if workplace == "museums_or_galleries" || workplace == "cinema" || workplace == "indoor_attraction"
+    render_terse_options = true
```

Comment:
it might be better to just list the options here

```
case workplace
when "museums_or_galleries", "cinema", "indoor_attraction"
  options = {
    "yes": "Yes",
    "no": "No",
  }
else
  options = {
    "yes": "Yes, I work at one of these exceptions",
    "no": "No, I do not work at one of these exceptions",
  }
end
```